qDict/src/main/res/layout/fragment_recent.xml

```diff
@@ -22,7 +22,7 @@
             android:gravity="start|center_vertical"
             android:text="@string/recent_lable"
             android:textColor="@color/mmt_black"
-            android:textSize="28.0sp" />
+            android:textSize="28.0dp" />
```

Comment:
Please mention the rationale for this change in the PR description, for future reference.

Reply:
OK, I would demonstrate this one. As you might see, before my fix, the string of "Recent words" overlaps "0 key word" behind, so I guess the text size of "Recent words" is so large that they overlap together. That is why I changed "sp" to "dp", because if we use "dp", the text size won't grow with the display size of mobile device. And after my fix, they are separated from each other. And I guess that is a fix. :)

Reply:
Or, in other words, DP means that the text is DPI-independent, scale-dependent; SP means that the text is DPI-independent and scale-independent. Here the text is large anyway, so we're fine having a scale-independent header.